ofnet.go

```diff
@@ -67,6 +67,15 @@ type OfnetDatapath interface {
 
 	//Delete uplink port
 	RemoveUplink(portNo uint32) error
+
+	// Add a service spec to proxy
+	AddSvcSpec(svcName string, spec *ServiceSpec) error
+
+	// Remove a service spec from proxy
+	DelSvcSpec(svcName string, spec *ServiceSpec)
+
+	// Service Proxy Back End update
+	SvcProviderUpdate(svcName string, providers []string)
```

Comment:
For this one, I don't think we can return any error because it is legal for a service spec and provider update to come out of order.